Assets/MRTK/Services/InputSimulation/SimulatedGestureHand.cs

```diff
@@ -47,6 +47,11 @@ public class SimulatedGestureHand : SimulatedHand
             MixedRealityInteractionMapping[] interactions = null)
                 : base(trackingState, controllerHandedness, inputSource, interactions)
         {
+            EnsureProfileSettings();
+            if (controllerHandedness == Handedness.Any)
+            {
+                SetupDefaultInteractions();
```

Comment:
This feels odd to me. The base class should handle this case. Possibly related to my other comment, if we're looking to add a new handedness to this class.

Reply:
Yeah I've modified these hacks to properly use a new controller input profile definition.